python/tvm/target/tag.py

```diff
@@ -57,14 +60,14 @@ def register_tag(name: str, config: Dict[str, Any], override: bool = False) -> T
             "arch": "sm_61",
         })
     """
-    return _ffi_api.TargetTagAddTag(name, config, override)
+    if hasattr(_ffi_api, "TargetTagAddTag"):
+        return _ffi_api.TargetTagAddTag(name, config, override)
+    return None
 
 
-# We do a round of tag listing on loading time to verify all tag information
-if hasattr(_ffi_api, "TargetTagAddTag"):
-    list_tags()
+list_tags()
 
-    register_tag("nvidia/gtx1080ti", config={
-        "kind": "cuda",
-        "arch": "sm_61",
-    })
+register_tag("nvidia/gtx1080ti", config={
+    "kind": "cuda",
+    "arch": "sm_61",
+})
```

Comment:
Per offline discussion, we maintain all tags in the C++ side to support pure C++ use cases. The Python API is only used for prototyping.

Reply:
I will add this sentence to the code comments.